sdk/storage/azure_storage_blob/src/clients/blob_client.rs

```diff
@@ -1,2 +1,66 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+
+use crate::{
+    models::BlobProperties, pipeline::StorageHeadersPolicy, BlobBlobClientGetPropertiesOptions,
+    BlobClientOptions, GeneratedBlobClient,
+};
+use azure_core::{credentials::TokenCredential, BearerTokenCredentialPolicy, Policy, Result, Url};
+use std::sync::Arc;
+
+pub struct BlobClient {
+    pub endpoint: Url,
+    pub container_name: String,
+    pub blob_name: String,
+    client: GeneratedBlobClient,
+}
+
+impl BlobClient {
```

Comment:
In other languages, I thought you can only get a `BlobClient` from a `ContainerClient` or something i.e., these aren't creatable. If so, they must be exported from both the crate root (re-exported) and from the `clients` module in which all clients - creatable or gettable - are exported.